packages/certs-v5/lib/endpoints.js

```diff
@@ -2,15 +2,6 @@
 
 const { checkPrivateSniFeature } = require('./features.js')
```

Comment:
I don't think this is needed anymore

Reply:
removed at 4f05dc2d7761d138506b5058cac080fa88bd37b8